app/GraphQL/Ecosystem/Mutations/Apps/AppSettingsMutation.php

```diff
@@ -25,4 +25,19 @@ public function saveSettings(mixed $root, array $req): mixed
 
         return $app->get($appSetting->name);
     }
+
+    /**
+     * saveAppSmtpSettings
+    */
+    public function saveAppSmtpSettings(mixed $root, array $req): mixed
+    {
+        $app = AppsRepository::findFirstByKey($req['id']);
```

Comment:
@FredPeal they cant send you the app key , its always the current app that's running based on the header key